utils_linux.go

```diff
@@ -296,6 +296,12 @@ func (r *runner) run(config *specs.Process) (_ int, retErr error) {
 			return -1, err
 		}
 	}
+	if r.notifySocket != nil {
+		if err = r.notifySocket.forward(process, detach); err != nil {
+			r.terminate(process)
```

Comment:
Wasn't just added a defer to do this before here? Do we need it again?

Reply:
Yes, I'll remove this terminate operation later. After each rebase, I should check it carefully.